docs/examples/201/front-door-with-webapplication-firewall/main.bicep

```diff
@@ -0,0 +1,176 @@
+param frontDoorName string = 'AzFd-TestingBicep-999'
+param frontDoorEnabledState bool = true
+param healthProbe1EnabledState bool = false
+param frontDoorWafNamePrefix string = 'AzFdWafTestingBicep'
+param frontDoorWafEnabledState bool = true
+param frontDoorWafMode string {
+  allowed: [
+    'Prevention'
+    'Detection'
+  ]
+  default: 'Prevention'
+}
+
+var frontDoorNameLower = toLower(frontDoorName)
+var backendPool1Name = concat(frontDoorNameLower, '-', 'backendPool', '1')
```

Comment:
use string interpolation instead of concat:

```
var backendPool1Name = '${frontDoorNameLower}-backendPool1'
```

Can you do this for all uses of `concat()`?

Reply:
Thanks. Improved now.
More changes to come when we have conditions in 0.3